goldens/public-api/common/common.d.ts

```diff
@@ -110,7 +110,7 @@ export declare class HashLocationStrategy extends LocationStrategy {
 
 export declare class I18nPluralPipe implements PipeTransform {
     constructor(_localization: NgLocalization);
-    transform(value: number, pluralMap: {
+    transform(value: number | null | undefined, pluralMap: {
```

Comment:
We could go crazy here and do something like:

```suggestion
    transform(value: number, ...): string;
    transform(value: null | undefined, ...): '';
    transform(value: number | null | undefined, ...): string;
```

Reply:
sure! :) should I go forward with that?

Reply:
In a way, it looks like a good idea, as it provides a type-level "documentation" of the behavior in these special cases.
OTOH, it looks like a strange commitment from the perspective of the API